bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/data/ModuleType.java

```diff
@@ -135,8 +135,9 @@
     public final List<String> extensions = new LinkedList<>();
     public final List<Class<? extends ChannelHelper>> channelHelpers;
     public final List<Class<? extends Capability>> capabilities;
+    public final String uid;
```

Comment:
Can you rename this to `thingTypeId` for clarity? By name it seems like `uid` is compared to `id` in method `from()`, which would be wrong, but actually it's the variable name which is wrong.